user/tests/wiring/i2c_master_slave/i2c_master/i2c_master.cpp

```diff
@@ -31,6 +31,9 @@ test(I2C_01_Master_Slave_Master_Variable_Length_Transfer)
         memcpy(I2C_Test_Tx_Buffer, MASTER_TEST_MESSAGE, sizeof(MASTER_TEST_MESSAGE));
         memcpy(I2C_Test_Tx_Buffer + sizeof(MASTER_TEST_MESSAGE), (void*)&requestedLength, sizeof(uint32_t));
 
+        // Exit sleep
```

Comment:
Add a test to make sure that the buffers are unaffected between sleep cycles.

Reply:
Added.